ocs_ci/ocs/ui/base_ui.py

```diff
@@ -509,29 +521,28 @@ class PageNavigator(BaseUI):
 
     """
 
-    def __init__(self, driver):
-        super().__init__(driver)
+    def __init__(self):
+        super().__init__()
         self.ocp_version = get_ocp_version()
+        self.running_ocp_semantic_version = version.get_semantic_ocp_running_version()
         self.ocp_version_full = version.get_semantic_ocp_version_from_config()
-        self.page_nav = locators[self.ocp_version]["page"]
         self.ocs_version_semantic = version.get_semantic_ocs_version_from_config()
         self.ocp_version_semantic = version.get_semantic_ocp_version_from_config()
-        self.running_ocp_semantic_version = version.get_semantic_ocp_running_version()
+
+        self.page_nav = locators[self.ocp_version]["page"]
+        self.generic_locators = locators[self.ocp_version]["generic"]
+        self.validation_loc = locators[self.ocp_version]["validation"]
+        self.dep_loc = locators[self.ocp_version]["deployment"]
```

Comment:
Why did you add this line to PageNav class?
```self.dep_loc = locators[self.ocp_version]["deployment"]```
it is problematic, because if I want to add only `add capacity` test on OCP4.14... the test will fail because locators[14]["deployment"] doesn't exist.

Reply:
this PR is for 4.13. It has all locators ready. 
Other than that this is artificial blocker caused because of separation locator dicts to the pages. I don't see a reason to not have all locators once we have 4.14.

Having all locators ready we will avoid skip_if conditions.

Reply:
@OdedViner 
fixed, locators extraction is safe with dict - reduce approach